daemon/cmd/daemon_main.go

```diff
@@ -752,6 +752,9 @@ func initializeFlags() {
 	flags.Bool(option.PProf, false, "Enable serving the pprof debugging API")
 	option.BindEnv(Vp, option.PProf)
 
+	flags.String(option.PProfAddress, defaults.PprofAddressAgent, "Address that the pprof listens on")
```

Comment:
Nit:

```suggestion
	flags.String(option.PProfAddress, defaults.PprofAddressAgent, "Address that pprof listens on")
```

Same here, might change it in the pprof port message as well.